v1/related.js

```diff
@@ -11,59 +11,89 @@ class Related {
         this._options = options;
     }
 
+    _relatedPageRedirect(hyper, domain, title) {
+        const params = `action=query&format=json&titles=${title}&redirects=1&converttitles=1`;
+        const apiURL = `https://${domain}/w/api.php?${params}`;
```

Comment:
Instead of building the URL here, you should be able to use the action API wrapper here. Example: https://github.com/wikimedia/restbase/blob/master/v1/lists.js#L142-L154

Reply:
Done.